khtools/compare_kmer_content.py

```diff
@@ -12,8 +12,10 @@
 # Divergence time estimates in millions of years
 # from http://www.timetree.org/ on 2019-08-26
 from .sequence_encodings import amino_keto_ize, \
-    weak_strong_ize, purine_pyrimidize, dayhoffize, dayhoff_v2_ize, hpize, \
-    botvinnikize
+    weak_strong_ize, purine_pyrimidize, encode_peptide
+
+MOLECULES_TO_COMPARE = 'aa20', 'dayhoff6', 'hp2', 'botvinnik8', 'aa9', \
```

Comment:
is there some notes in regard to what this would mean? I think the number followed by Dayhoff is how many base pairs form an encoding, it would be good to add notes for the future? like I don't know what gbmr4 is. Looks like the Peterson paper mentions it but not clear what they stand for?